### PR TITLE
Move unpacking eggs bundle to portia-entrypoint

### PR DESCRIPTION
**Pull request by vshlapakov**
The PR merges eggbased-entrypoint to portia-entrypoint and adds downloading/unpacking eggs bundle. It also changes a location of packed eggs from /app to /tmp/packed-eggs as we don't have access to modify /app from the entrypoint. Additional change is to load built-in addons eggs.

Review, please.

### CODE REVIEW

File: portia-entrypoint
Comment (vshlapakov) [inline portia-entrypoint]: Please, note that I dropped logic for `scrapy-list` here, looks like it's useless as for now we don't use portia-stack with builder.

File: portia-entrypoint
Comment (chekunkov) [inline portia-entrypoint]: if pypath is not empty and doesn't end with colon you'll get a malformed pythonpath here. also I think u may want dash eggs to be at the beginning of the path, not appended to the end

File: portia-entrypoint
Comment (chekunkov) [inline portia-entrypoint]: hm, why we don't have permission to remove from tmp?

**Review by chekunkov — CHANGES_REQUESTED** [review]

File: portia-entrypoint
Comment (vshlapakov) [reply portia-entrypoint]: Right as now we unpack eggs bundle to tmp 👍

File: portia-entrypoint
Comment (vshlapakov) [inline portia-entrypoint]: We also have to load addons eggs built-in to the stack, the only question is whether we should allow to replace the addons eggs with custom eggs or not (it's defined by precedence of the paths here). Assuming that we're moving away from eggs completely, I think it's fine to keep it as-is, but let me know if you disagree

File: portia-entrypoint
Comment (chekunkov) [inline portia-entrypoint]: is this note about "scrapy list" still relevant?

**Comment by vshlapakov** [comment]
@chekunkov Let me know if there's anything else awaiting for me, looks like everything is covered

**Review by chekunkov — APPROVED** [review]

**Comment by chekunkov** [comment]
@vshlapakov lgtm, do we need to wait for the new entrypoint to be released before we merge this pr, or this one is still compatible with current one?

**Comment by vshlapakov** [comment]
@chekunkov Have just checked it in a case, the PR is compatible with current entrypoint, the only drawback I should mention about is that some work is done twice, for now eggs bundle is:
- downloaded with Mesos to sandbox dir & unpacked with kumo-entrypoint to /app
- downloaded with portia-entrypoint to /tmp/ & unpacked to /tmp/unpacked-eggs

But it looks like a necessary transition state, when portia-stack is updated we're able to:
- stop providing eggs bundle with Mesos
- replace kumo-entrypoint with new version when it's ready

**Comment by chekunkov** [comment]
In this case we can merge and try it out on prod

**Pull request merged**
